### ISSUE TITLE
Attention to Draco and Xenomorph

### ISSUE THREAD

**Issue by chaosvolt** [chaosvolt/MST_Extra_Mod#3]
@DracoGriffin, @Xenomorph-III

Regarding issue: https://github.com/CleverRaven/Cataclysm-DDA/issues/24511

You're pinging two users who are blocked on CDDA's repo. Neither @DangerNoodle nor I will receive your pings on our notifications section. Blocked users also won't be able to respond in the repo itself, and they definitely can't fix other people's negligence via submitting PRs.

In the absence of a mod's original authors and maintainers, other people have to actually do it. Side note, I'm still of the opinion that, if a PR author doesn't at least make a token effort to check the official mods, they have no excuse. It's good practice and avoids generating more work in the future. The "break it now, fix it later" attitude is what created a brief "holy shit we need more devs" panic a while back/

But then again, a lot of my arguments with the devs back in the day could be summed up as me getting salty over people creating more busywork for other contributors due to laziness in their own PRs, so...

tl;dr fix it yourself, Dangernoodle probably will be fine with it (if they ever weigh in on it at all, they don't seem to have been active), that is all.
